src/amo/components/ContributeCard/index.js

```diff
@@ -71,6 +84,14 @@ export const ContributeCardBase = ({ addon, i18n }: Props) => {
       break;
   }
 
+  const onButtonClick = (event: SyntheticEvent<HTMLAnchorElement>) => {
+    _tracking.sendEvent({
+      action: addon.slug,
```

Comment:
This should be the `label` (and we use the `name` usually). Actions should be `click`, etc.

Reply:
Although the GA documentation (and the name of the property) suggest that `action` should be the action (e.g., `click`), that's not actually how we use that property in most cases. We often include the "action" in the `Event Category` (e.g., "AMO Primary Hero Clicks", "AMO Addon Installs Started"), and then we use the `Event Action` property to further define what the event applied to. E.g., for "AMO Primary Hero Clicks" the `action` is the destination URL, and for "AMO Addon Installs Started" the `action` is the `addonType`. 

In this case we want to track two additional pieces of data, the add-on for which the Contribute button was clicked, and the destination URL. So I have assigned one of those to `action` and one to `label`. I chose to use the add-on's slug, rather than the name, as it seemed like that would work better for generating statistics, as the slug is less likely to change than the name.

I can reverse my use of the properties to put the target URL in `action` and the add-on's slug in `label`, as that does seem to be somewhat consistent with what we're doing in other events, but tbph there doesn't seem to be a whole lot of consistency in terms to of the data we send with different events.